source/MRMesh/MRPolylineTrimWithPlane.h

```diff
@@ -39,11 +39,18 @@ MRMESH_API void trimWithPlane( Polyline3& polyline, const Plane3f& plane, const
 [[deprecated]] MRMESH_API void dividePolylineWithPlane( Polyline3& polyline, const Plane3f& plane, const DividePolylineParameters& params = {} );
 
 /// This function cuts polyline with a plane
-/// \return Edge segments that are closer to the plane than \param eps
+/// \details It cuts only edges, one end of which is in the negative half-space, and the other is not
+/// \return Edge segments intersecting with a plane
 /// \param polyline Input polyline that will be cut by the plane
 /// \param plane Input plane to cut polyline with
-/// \param eps Maximal distance from the plane
-/// \param positiveEdges Edges in a positive half-space relative to the plane (only adds bits to the existing ones)
-MRMESH_API std::vector<EdgeSegment> extractSectionsFromPolyline( const Polyline3& polyline, const Plane3f& plane, float eps, UndirectedEdgeBitSet* positiveEdges = {} );
```

Comment:
Please keep old function, since it is used in SectionPlugin, and created new one `extractSectionFromPolyline`, which
1) does not take eps on input
2) returns std::vector<EdgePoint>